app/controllers/users_controller.rb

```diff
@@ -58,13 +58,12 @@ def user_params
   def correct_user
     return if @user == current_user
 
-    redirect_to(root_url,
-                status: :see_other)
+    redirect_to root_url, status: :see_other
   end
 
   # Confirms an admin user.
   def admin_user
-    redirect_to(root_url, status: :see_other) unless current_user.admin?
+    redirect_to root_url, status: :see_other unless current_user.admin?
```

Comment:
```suggestion
    redirect_to(root_url, status: :see_other) unless current_user.admin?
```
case này phải có ngoặc nhé nếu k sẽ dẹo